components/gitpod-db/src/typeorm/entity/db-code-sync-resource.ts

```diff
@@ -17,14 +17,16 @@ export const enum SyncResource {
     Settings = 'settings',
     Keybindings = 'keybindings',
     Snippets = 'snippets',
+    Tasks = 'tasks',
     Extensions = 'extensions',
-    GlobalState = 'globalState'
+    GlobalState = 'globalState',
 }
-export const ALL_SYNC_RESOURCES: SyncResource[] = [SyncResource.Settings, SyncResource.Keybindings, SyncResource.Snippets, SyncResource.Extensions, SyncResource.GlobalState];
+export const ALL_SYNC_RESOURCES: SyncResource[] = [SyncResource.Settings, SyncResource.Keybindings, SyncResource.Snippets, SyncResource.Tasks, SyncResource.Extensions, SyncResource.GlobalState];
 
 export interface IUserDataManifest {
-    latest: Record<ServerResource, string>
-    session: string;
+	readonly latest?: Record<ServerResource, string>;
+	readonly session: string;
+	readonly ref: string;
```

Comment:
Exactly as @iQQBot said, I need to somehow get a `ref` and add it to here when syncing, haven't looked into it yet.